be2-scala/src/main/scala/ch/epfl/pop/model/objects/Channel.scala

```diff
@@ -0,0 +1,20 @@
+package ch.epfl.pop.model.objects
+
+import java.util.Base64
+
+object Channel {
+  type Channel = String
+
+  val rootChannelPrefix: String = "/root/"
+
+  /**
+   * Extract the sub-channel (laoId) from a channel name
+   *
+   * @param channel full channel name
+   * @return an array of bytes corresponding to the decoded sub-channel name or None if an error occurred
+   */
+  def decodeSubChannel(channel: Channel.Channel): Option[Array[Byte]] = channel match {
+    case _ if channel.startsWith(rootChannelPrefix) => Some(Base64.getDecoder.decode(channel.getBytes)) // TODO may throw
+    case _ => None
+  }
```

Comment:
do we even need this? 
I don't think we have this logic anywhere else, on the other hand we do need to do the reverse, from a lao id getting the sub-channel

Reply:
I'm not sure yet how useful the "Channel" case class is. Last semester, we emitted the possibility of having sub-sub-channels (e.g. root/xabc/def). Is this still up-to-date?

Reply:
Yes, students need this for the e-voting project :)